checkov/terraform/module_loading/loaders/local_path_loader.py

```diff
@@ -10,7 +10,8 @@ def __init__(self) -> None:
         self.is_external = False
 
     def _is_matching_loader(self) -> bool:
-        return self.module_source.startswith("./") or self.module_source.startswith("../") or self.module_source.startswith(self.current_dir)
+        return self.module_source.startswith("./") or self.module_source.startswith("../") \
+               or self.module_source.startswith(self.current_dir) or self.module_source.startswith('/')
```

Comment:
`self.module_source.startswith(self.current_dir)`

This might cause problems. If the current directory is called `s3` or `https` or something like that, it would conflict with other loaders. What case is that condition trying to cover?

Reply:
Sorry, `current_dir` might be the full path? If so, ignore me.